examples/server/server.c

```diff
@@ -276,6 +280,114 @@ static int TestEmbedSendTo(WOLFSSL* ssl, char *buf, int sz, void *ctx)
 
 #endif /* WOLFSSL_DTLS */
 
+#ifdef WOLFSSL_WOLFSENTRY_HOOKS
```

Comment:
Please format the lines to avoid longer than 80 chars. At 308, 342, 350, 359, 373

Reply:
:heavy_check_mark: